modules/sandboxed-containers-check-nodes.adoc

```diff
@@ -0,0 +1,32 @@
+// Module included in the following assemblies:
+//
+// * sandboxed_containers/deploying-sandboxed-container-workloads.adoc
+
+:_module-type: PROCEDURE
+[id="sandboxed-containers-check-nodes_{context}"]
+= Checking the eligibility of cluster nodes to run Kata containers
```

Comment:
I would use the product name.

= Checking the eligibility of cluster nodes to run {sandboxed-containers-first}

Reply:
The label will be either created by NFD or manually by the administrator. Consequently, I think we can remove the follow-on content and replace it with the content under "Deploying the check node feasibility feature on all eligible nodes"
@pmores @jfreiman wdyt ?

Reply:
@bpradipt I agree, it would be better not to have multiple sections of text dealing with the same topic, slightly differently.  In general, I'm happy with any solution as long as it's clear that the user has basically three options:
- do nothing
- add the labels themselves, manually
- install NFD
and that the user understands the consequences of each of them.

Reply:
OK yes. I think I ewas overthinking it. I'll remove the extraneous topics.